### PR TITLE
Add tool on creating issue/pr templates

### PR DESCRIPTION
**Pull request by ssh24**
A tool to create issue and pr templates on all loopback repos.

connect to https://github.com/strongloop-internal/scrum-apex/issues/293

### CODE REVIEW

File: issue-pr-templates/bin/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  https://strongloop.com/api-connect-faqs/
+  https://strongloop.com/node-js/subscription-plans/
+-->
+
+# Description/Steps to reproduce
+
+<!--
+If feature: A description of the feature
+If bug: Steps to reproduce
+-->
```
Comment (dhmlau) [inline issue-pr-templates/bin/ISSUE_TEMPLATE.md]: Thinking out loud and open for discussion... Now that we're more strict about users providing a sandbox for us to reproduce the problem, should we emphasize in here?  something like: 
- if you don't provide this to us, we won't look at it (of course with nicer wording) or expect delays
- And if they think it's very easy to reproduce and there's no need for a sandbox, tell us why.

Comment (ssh24) [inline issue-pr-templates/bin/ISSUE_TEMPLATE.md]: I did add a header to link sandbox repo couple of lines [down](https://github.com/strongloop-internal/loopback-scripts/pull/6/files#diff-a8300ae96c4fcdba614fc3f7a8970546R17)

Comment (dhmlau) [inline issue-pr-templates/bin/ISSUE_TEMPLATE.md]: What I meant was we should be more explicit in telling people up front that we need the sandbox from them for reproducing the issue or we won't look at it.

File: issue-pr-templates/lib/util.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+const assert = require('assert');
+const Octokat = require('octokat');
+
+module.exports = Util;
+
+function Util(creds) {
+  this.octo = new Octokat(creds);
+};
+
```
Comment (b-admike) [inline issue-pr-templates/lib/util.js]: Add API docs for these utility functions, so that people can utilize them in the future :)

Comment (ssh24) [inline issue-pr-templates/lib/util.js]: Done.

**Pull request merged**